apis/BaseContentEndpoint.js

```diff
@@ -48,80 +47,49 @@ export const formatBaseContentQuery = (query, method) => {
   return query;
 };
 
-export default class BaseContentEndpoint extends BaseEndpoint {
-  static move = BaseEndpoint.notImplemented('move');
-  static copy = BaseEndpoint.notImplemented('copy');
-  static put = BaseEndpoint.notImplemented('put');
-
-  static formatQuery(query, method) {
-    if (method !== 'patch') {
-      query.feature = 'link_preview';
-      query.sr_detail = 'true';
-    }
+export const  patchPath = () => {
+  return 'api/editusertext';
+};
 
-    if (method === 'del') {
-      query._method = 'post';
-    }
+export const  deletePath = () => {
+  return 'api/del';
+};
 
-    return query;
+export const  patch = (apiOptions, data) => {
+  if (!data) {
+    throw new NoModelError('/api/editusertext');
   }
 
-  static path(method, query={}) {
-    if (method === 'get') {
-      return this.getPath(query);
-    } else if (method === 'post') {
-      return this.postPath(query);
-    } else if (method === 'patch') {
-      return this.patchPath(query);
-    } else if (method === 'del') {
-      return this.deletePath(query);
-    }
-  }
+  const promises = [];
 
-  static patchPath () {
-    return 'api/editusertext';
-  }
+  Object.keys(data).map(k => {
```

Comment:
I've been using the lodash methods in my own code, wouldn't be a bad idea if everyone can palette using them. Tbh I was just moving code around to get rid of the class (that removed ~ 2KB iirc, but the model took that space back)